crates/db_views_actor/src/structs.rs

```diff
@@ -20,6 +20,7 @@ pub struct CommunityBlockView {
 pub struct CommunityFollowerView {
   pub community: CommunitySafe,
   pub follower: PersonSafe,
+  pub pending: Option<bool>,
```

Comment:
I think we can make a migration to make this value mandatory (and set true where it is currently null). Having to check for null in clients would be annoying.

Reply:
I can add that to this today.